src/ucs/memory/rcache.c

```diff
@@ -662,9 +663,9 @@ static UCS_CLASS_INIT_FUNC(ucs_rcache_t, const ucs_rcache_params_t *params,
         goto err_free_name;
     }
 
-    ret = pthread_spin_init(&self->inv_lock, 0);
+    ret = ucs_spinlock_init(&self->inv_lock);
```

Comment:
ret->ucs_status
compare to UCS_OK
move ucs_error inside ucs_spinlock_init

Reply:
fixed